src/sections/messages/index.js

```diff
@@ -1,9 +1,146 @@
+import './styles.css';
+
+const contacts = [
+    {
+        name: 'Saray',
+        lastDateSeen: '8:30 am',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is A rainbow i a meteorological phenomenon that is something',
+        avatarSrc: '/assets/images/user2.jpg',
+        isActive: true,
+    },
+    {
+        name: 'Leo',
+        lastDateSeen: '5:30 pm',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is something',
+        avatarSrc: '/assets/images/user1.jpg',
+    },
+    {
+        name: 'Rey',
+        lastDateSeen: '10:15 pm',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is something',
+        avatarSrc: '/assets/images/user3.jpg',
+    },
+    {
+        name: 'Jose',
+        lastDateSeen: '8:45 am',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is something',
+    },
+    {
+        name: 'Juan',
+        lastDateSeen: 'yesterday',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is something',
+    },
+    {
+        name: 'Tahimi',
+        lastDateSeen: 'yesterday',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is something',
+    },
+    {
+        name: 'Pepe',
+        lastDateSeen: 'yesterday',
+        lastMessage: 'A rainbow i a meteorological phenomenon that is something',
+    },
+];
+
+const texts = [
+    {
+        avatarSrc: '/assets/images/user2.jpg',
+        message: 'I have a problem with a topup',
+        dateSend: '1:06 PM',
+    },
+    {
+        avatarSrc: '/assets/images/user4.jpg',
+        message: 'Verify the account',
+        isUser: true,
+        checked: true,
+        dateSend: '1:07 PM',
+    },
+    {
+        avatarSrc: '/assets/images/user2.jpg',
+        message: 'Already all is ok, thanks',
+        dateSend: '1:19 PM',
+    },
+    {
+        avatarSrc: '/assets/images/user4.jpg',
+        message: 'ok',
+        isUser: true,
+        dateSend: '1:22 PM',
+    },
+];
+
+export default class Messages extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchTerm: '',
+        };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    getContacts() {
+        const { searchTerm } = this.state;
+        return filter(searchTerm, contacts);
+    }
+
+    handleOnChange(event) {
+        this.setState({ searchTerm: event.target.value });
+    }
+
+    render() {
+        const { searchTerm } = this.state;
+        return (
+            <div className="react-rainbow-admin-messages">
+                <div className="react-rainbow-admin-messages_contacts-container">
+                    <Input
+                        className="react-rainbow-admin-messages_contacts-search"
+                        label="Find contact"
+                        type="search"
+                        hideLabel
+                        placeholder="Find contact"
+                        aria-label="Find contact"
```

Comment:
now this is not being passed to input element, the Input component does not expect any aria-label prop